docs/io/fits/index.rst

```diff
@@ -530,6 +530,26 @@ encapsulate it in an :class:`HDUList` object first.
 Creating a New Table File
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. note::
+
+    If you want to create a simple binary FITS table with no other HDUs,
```

Comment:
If you want to see examples of ASCII FITS tables, it's an option when downloading a table from Vizier